.gitignore

```diff
@@ -4,7 +4,7 @@
 
 .vscode
 .tox
-.pytest_cache
+.*_cache
```

Comment:
**To the reviewer:**

This should cover **.mypy_cache** and **.pytest_cache**.